ci-operator/jobs/openshift/installer/openshift-installer-release-4.12-presubmits.yaml

```diff
@@ -3054,6 +3054,79 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-vsphere-upi,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
Generally I have been trying to avoid having always_run: true except for very general jobs. Could we have this run only when changes are in files with vsphere in the path?

Reply:
good catch, the intention is for this to run on demand.